### ISSUE TITLE
From section (p. 73): Adding the Stratospheric Construct Library

### ISSUE THREAD

**Issue by ekimike** [stratospheric-dev/cdk-constructs#34]
Not sure if the book needs to be update to new dependency version from:

```java
<dependency> 
<groupId>dev.stratospheric</groupId> 
<artifactId>cdk-constructs</artifactId> 
<version>0.0.27</version>
</dependency>
```

to

```java
<dependency> 
<groupId>dev.stratospheric</groupId> 
<artifactId>cdk-constructs</artifactId> 
<version>0.1.0</version>
</dependency>
```

After the change above it compiles, but still, there is some incongruence after running the dependency tree:

```
Dependency convergence error for software.amazon.awscdk:aws-cdk-lib:2.5.0 paths to dependency are:
+-com.myorg:first_cdk_app:0.1
  +-dev.stratospheric:cdk-constructs:0.1.0
    +-software.amazon.awscdk:aws-cdk-lib:2.5.0
and
+-com.myorg:first_cdk_app:0.1
  +-software.amazon.awscdk:aws-cdk-lib:2.0.0

[WARNING] Rule 0: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
Failed while enforcing releasability the error(s) are [
Dependency convergence error for software.amazon.awscdk:aws-cdk-lib:2.5.0 paths to dependency are:
+-com.myorg:first_cdk_app:0.1
  +-dev.stratospheric:cdk-constructs:0.1.0
    +-software.amazon.awscdk:aws-cdk-lib:2.5.0
and
+-com.myorg:first_cdk_app:0.1
  +-software.amazon.awscdk:aws-cdk-lib:2.0.0
]
```

You can reproduce the above issue by adding:

```
<plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-enforcer-plugin</artifactId>
                <version>1.4.1</version>
                <configuration>
                    <rules><dependencyConvergence/></rules>
                </configuration>
 </plugin>
```

And running:

```
mvn enforcer:enforce
```

It seems you are ahead of AWS

**Comment by BjoernKW** [comment]
Thanks for bringing this up. With the latest releases of the CDK constructs library (0.1.0) and the ebook (1.4) we've updated and synchronised all references to CDK libraries to the latest available version (2.8.0 for `software.amazon.awscdk.aws-cdk-lib` and 10.0.38 for `software.constructs.constructs` at release time).

I've just run `mvn enforcer:enforce` as per your suggestion and everything looks fine right now. Could you please check again if the error persists with the most recent Stratospheric CDK Constructs version and the source code from the latest ebook release?

**Comment by ekimike** [comment]
Hi, all looks fine now. Thanks
